models/pathology_tumor_detection/docs/README.md

```diff
@@ -39,6 +39,19 @@ The training was performed with the following:
 - Loss: BCEWithLogitsLoss
 - Whole slide image reader: cuCIM (if running on Windows or Mac, please install `OpenSlide` on your system and change `wsi_reader` to "OpenSlide")
 
+By setting the `"pretrained"` parameter of `TorchVisionFCModel` in the config file to `true`, ImageNet pre-trained weights will be used for training. Please note that these weights are for non-commercial use. Each user is responsible for checking the content of the models/datasets and the applicable licenses and determining if suitable for the intended use. In order to use other pretrained weights, you can use `CheckpointLoader` in train handlers section as the first handler:
```

Comment:
Hi @zephyrie , could you please also help to review this part? Thanks!

Reply:
This looks good! I might suggest a header of "### Pretained Weights" so it's easier to see.  Also, I think it would make sense to move up the "Notes on Determinism" section to be under training configuration rather than MONAI Bundle Commands. You could just add it to the end of the "Training Configuration" section, that way all of the training caveats are in a single place.